src/ser.rs

```diff
@@ -309,6 +310,22 @@ impl<'s> ser::Serializer for &'s Serializer {
         }
     }
 
+    fn serialize_i128(self, v: i128) -> Result {
+        if self.serialize_large_number_types_as_big_ints {
+            Ok(JsValue::from(v))
+        } else {
+            Err(Error::custom("i128 is not supported"))
```

Comment:
Hm, shouldn't this fallback to JS number when option isn't activated? I'm of two minds on this, perhaps it's fine for it to error out, but then it would be good to at least expand the error message to clarify that it's supported, just needs to be enabled.

Reply:
Good point. The current behavior is to error so I was going to uphold that. In addition, I don't think there are many cases where it would be a good idea to convert a 128-bit integer to a 53 bit one (Js Number).

Reply:
Yeah erroring is fine if that's current behaviour, just saying the error message probably needs to be clearer and nudge the user in the right direction.

Reply:
Even something like "i128 is supported only in `serialize_large_number_types_as_big_ints` mode" would already go a long way.

Reply:
My bad fixed it in one place and not the other